Create ui-c8y-1020-28-8-no-possibility-to-delete-devices-in-child-devices-grid.md (#2473)
SoftwareAG/c8y-docs: content/change-logs/device-management/ui-c8y-1020-28-8-no-possibility-to-delete-devices-in-child-devices-grid.md

---
date: ""

> title: Enable deletion of child devices in Device Management

Reviewer: ```suggestion
title: Enabled deletion of child devices from the Child devices tab
```

> title: No possibility to delete devices in Child devices grid

Author: The Claude AI suggested to use the following title:
```suggestion
title: Enable deletion of child devices in the device management
```

> title: No possibility to delete devices in Child devices grid

Reviewer 2: ```suggestion
title: Enable deletion of child devices in Device Management
```

product_area: Device management & connectivity
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-3951
version: 1020.28.8
---

> No possibility to delete devices in Child devices grid

Reviewer 2: ```suggestion
In Device Management, the action to delete child devices from the "Child devices" tab was removed, requiring users to navigate to the device details page for deletion. With this change, child devices can now be directly deleted from the "Child devices" grid view.
```

> No possibility to delete devices in Child devices grid

Author: The Claude AI suggested to use the following body:
```suggestion
In the device management, it was previously not possible to delete child devices from the grid view, requiring users to navigate to the device details page for deletion. With this change, child devices can now be directly deleted from the grid view in the same way as regular devices. This improvement simplifies the device management process and saves time for users with large device hierarchies.
```

> In Device Management, the action to delete child devices from the "Child devices" tab was removed, requiring users to navigate to the device details page for deletion. With this change, child devices can now be directly deleted from the "Child devices" grid view.

Reviewer: ```suggestion
In the Device Management application, child devices could not be deleted from the **Child devices** tab. Instead, users had to to navigate to the device details for deletion. With this change, child devices can now be directly deleted from the **Child devices** tab.
```
